src/aotcompile.cpp

```diff
@@ -258,7 +258,7 @@ static void jl_ci_cache_lookup(const jl_cgparams_t &cgparams, jl_method_instance
 // all reachable & inferrrable functions. The `policy` flag switches between the default
 // mode `0`, the extern mode `1`, and imaging mode `2`.
 extern "C" JL_DLLEXPORT
-void *jl_create_native_impl(jl_array_t *methods, const jl_cgparams_t *cgparams, int _policy)
+void *jl_create_native_impl(jl_array_t *methods, void *ctxt, const jl_cgparams_t *cgparams, int _policy)
```

Comment:
We will need a extern variant of this function, that takes a `LLVMContextRef` and unwraps that and then calls the internal variant (which shan't be exported).

Reply:
Does this also apply to the other DLL exported functions?

Reply:
They are simply reinterpret calls (https://llvm.org/doxygen/namespacellvm.html#a8f52f7baf3c7cf6519b12ca5ea8c6690), so we can just make them the primary interface for extern "C" without any extra overhead